python/dolfinx/fem/petsc.py

```diff
@@ -1118,3 +1126,94 @@ def interpolation_matrix(space0: _FunctionSpace, space1: _FunctionSpace) -> PETS
         Interpolation matrix.
     """
     return _interpolation_matrix(space0._cpp_object, space1._cpp_object)
+
+
+def copy_vec_to_function(
+    x: PETSc.Vec,
+    u: dolfinx.fem.Function,
```

Comment:
No need to pass a `dolfinx.fem.Function` here, only  PETSc vec from it is needed.